resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+restartfile=/opt/puppetlabs/server/data/<%= EZBake::Config[:real_name] %>/restartcounter.txt
+cur=$(head -n 1 $restartfile)
+initial=$cur
+timeout="300"
+
+if [ -e $restartfile ];  then
+    ${JAVA_BIN} ${JAVA_ARGS} -Djava.security.egd=/dev/urandom -cp ${INSTALL_DIR}/<%= EZBake::Config[:uberjar_name] %> clojure.main -m <%= EZBake::Config[:main_namespace] %> --config "${CONFIG}" -b "${BOOTSTRAP_CONFIG}" &
+    while [ "$cur" == "$initial" ] ;do
+        sleep 0.1
+        cur=$(head -n 1 $restartfile)
+
+        ((timeout--))
+        if [ "$timeout" = 0 ]; then
+            echo "Startup timed out"
+            exit 1
+        fi
+    done
+else
+    echo "Restart file does not exist"
```

Comment:
Same question here for @shrug / @underscorgan / @haus as what I asked in the other file. I don't want to keep that TCP port loop around, so I kind of think we should just say that no one should ever use this start script unless they've turned on the restart file configuration, and so it's desirable to fail hard right here.